src/components/CommentInfo/CommentInfo.jsx

```diff
@@ -1 +1,15 @@
-export const CommentInfo = () => <>Put the comment here</>;
+import React from 'react';
+import './CommentInfo.scss';
+
+export const CommentInfo = ({ comment }) => (
+  <div className="CommentInfo">
+    <div className="CommentInfo__title">
+      <strong className="CommentInfo__name">{comment.name}</strong>
+      {' by '}
+      <a className="CommentInfo__email" href={`malito:${comment.email}`}>
```

Comment:
There is a typo in the href attribute. It should be 'mailto:' instead of 'malito:'.